sys/src/nix/k10/multiboot.c

```diff
@@ -55,6 +55,16 @@ struct MMap {
 	u32int	type;
 };
 
+void
+multibootmemassert(u32int pmbi)
+{
+	Mbi *mbi;
+
+	mbi = KADDR(pmbi);
+	if ((mbi->memupper << 10) + (1<<20) < TMFM)
```

Comment:
Geez. MiB is defined here:
sys/src/9/pc/etherm10g.c:#define MiB            1048576u                /* Mebi 0x0000000000100000 */

but we all know what 1 << 20 is so ... ok.